python/pyspark/mllib/regression.py

```diff
@@ -193,18 +198,28 @@ def train(cls, data, iterations=100, step=1.0, miniBatchFraction=1.0,
               initialWeights=None, regParam=0.0, regType=None, intercept=False,
               validateData=True):
         """
-        Train a linear regression model on the given data.
-
-        :param data:              The training data.
-        :param iterations:        The number of iterations (default: 100).
+        Train a linear regression model with no regularization using
```

Comment:
This should not say "with no regularization."   It looks like this is a difference between the Python & Scala APIs.